### DISCUSSION TITLE
Change from Feather to Lucide

### DISCUSSION THREAD

**Discussion by RobbiNespu** [athul/archie#26] [Ideas]
web: https://lucide.dev 
repo: https://github.com/lucide-icons/lucide

seem Feather are not active anymore, it would be best to use it Lucide as replacement

**Comment by athul** [comment]
Seems like they aren't. After reading the reason for Lucide, I would love to switch to this but https://github.com/lucide-icons/lucide/issues/94

This seems a bit doubtful. Else would love to switch to this

**Reply by g-pacheco** [reply]
Just would like to add that they also do not have Academic icons.
It seems that people do not like academics :(
